src/AzureAuth/Commands/Ado/CommandPat.cs

```diff
@@ -25,6 +25,7 @@ namespace Microsoft.Authentication.AzureAuth.Commands.Ado
     /// An ADO Command for creating or fetching, and returning Azure Devops PATs.
     /// </summary>
     [Command("pat", Description = "Create and cache Azure Devops Personal Access Tokens (PATs) using encrypted local storage.")]
+    [Subcommand(typeof(Pat.CommandScopes))]

```

Comment:
Just out of curiosity, what is the purpose of adding this change? Was this a user feedback or how did you come up with solving this scenario and extending to a new subcommand

Reply:
I've run into this problem a few times, and again this morning while showing another engineer the command, but then not knowing exactly what the scope was that we needed and couldn't find the docs for it either :(. But I know we had validation of them, so it'd be nice if we could list them. Listing valid options is a very common action in this kind of scenario (e.g. `nvm ls-remote`, `rustup target list`, etc)